libs/block-scripts/shared/generate-dist-block-metadata.js

```diff
@@ -41,6 +41,16 @@ export const generateDistBlockMetadata = async (extra) => {
     ...extra,
   };
 
+  /* @todo - deprecate support for the old `schema` definitions and migrate to this being the primary implementation */
```

Comment:
Let's just delete `schema: "block-schema.json"` now – we don't support it

Reply:
[Remove outdated block-schema.json reference](https://github.com/blockprotocol/blockprotocol/pull/1158/commits/aff5a04fa9cee68ba4203d0f312d0779a958a6fd)

Note that I left the TODO. Can we completely drop `blockprotocol.schema` or will that break existing blocks or the hub or anything?

Reply:
It won't break anything, it's only read by `block-scripts`

Reply:
Well it will break block building if someone updates their `block-scripts` version and doesn't update their `package.json`. But I guess trying to support the legacy `package.json` format for that doesn't matter anyway as `codegen` will break.

Will remove the TODO

Reply:
Yes – this is why I suggest updating the error messagr from `block-scripts` to explain (or point to docs that do)

Reply:
See [Replace schema with blockEntityType and inject that into codegen …](https://github.com/blockprotocol/blockprotocol/pull/1158/commits/d8129045ce1ba4622c0b3c3d3cb191ec276698be) for updated code around this

I'm working on docs at the moment